lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -56,7 +56,6 @@ class CategoryRenderer {
 
     // Append audit details to header section so the entire audit is within a <details>.
     const header = /** @type {!HTMLDetailsElement} */ (this._dom.find('.lh-score__header', tmpl));
-    header.open = audit.score < 100; // expand failed audits
```

Comment:
looks like debugText is printed even if the header isn't `open`?

this header only reveals helpText and any `details` stuff?